lib/populate.js

```diff
@@ -0,0 +1,11 @@
+// populates missing values
+module.exports = populate;
+
+function populate(dst, src) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['populate' is defined but never used.](https://www.codacy.com/app/form-data/form-data/file/875240364/issues/source?bid=2805153&fileBranchId=2825813#l4)